installer/build-mac-dmg.sh

```diff
@@ -52,8 +53,11 @@ if [ -d "$OS_PATH/MacOS/python3.6" ]; then
   mv "$OS_PATH/MacOS/python36.zip" "$OS_PATH/Resources/python36.zip"; ln -s "../../Resources/python36.zip" "$OS_PATH/MacOS/lib/python36.zip";
 fi
 
+echo "Loop through bundled files and sign all binary files"
+find "build" \( -iname '*.dylib' -o -iname '*.so' \) -exec codesign -s "OpenShot Studios, LLC" --timestamp=http://timestamp.apple.com/ts01 --entitlements "installer/openshot.entitlements" --force "{}" \;
```

Comment:
This resigns all *.dylib and *.so files with a timestamp, since the --deep option ignores files that are already signed... and thus, forces a failed notarization.